src/utils.js

```diff
@@ -48,7 +48,11 @@ export function mapPathString(nodePath, state) {
     return;
   }
 
-  const modulePath = getRealPath(nodePath.node.value, state);
+  const sourcePath = nodePath.node.value;
+  const currentFile = state.file.opts.filename;
+  const opts = state.opts;
+
+  const modulePath = resolvePath.call(state, sourcePath, currentFile, opts);
```

Comment:
I was thinking of calling it with `resolvePath(sourcePath, currentFile, state.normalizedOpts)`.

And then the function we export in `index.js` would be a cunstom function doing:
```js
 export { 
  resolvePath: (sourcePath, currentFile, opts) => {
    const normalizedOpts = normalizeOptions(currentFile, opts);
    return resolvePath(sourcePath, currentFile, normalizedOpts);
  }
 };